Build/build.ps1

```diff
@@ -4,7 +4,7 @@ if ([string]::IsNullOrEmpty($assemblyVersion)) {$assemblyVersion = "0.0.1"}
 if ([string]::IsNullOrEmpty($nugetVersion)) {$nugetVersion = "0.0.1"}
 
 $vswhere = ${Env:\ProgramFiles(x86)} + '\Microsoft Visual Studio\Installer\vswhere'
-$msbuild = &$vswhere -products * -requires Microsoft.Component.MSBuild -latest -find MSBuild\**\Bin\MSBuild.exe
+$msbuild = &$vswhere -products * -requires Microsoft.Component.MSBuild -latest -find MSBuild\**\Bin\MSBuild.exe -prerelease
```

Comment:
I am not happy with this -prerelease thing. Why is this required? Do we maybe need to wait for a newer release of .NET Core?

Reply:
I needed this because I'm using only preview version of Visual Studio on my machine. We can remove this if it bothers you. It's not required for this PR.

Reply:
Yes please remove.

Reply:
Done